app/src/main/java/com/github/se/orator/ui/overview/SpeakingPracticeModule.kt

```diff
@@ -78,7 +73,7 @@ fun SpeakingPracticeModule(
                     Icon(
                         imageVector = Icons.AutoMirrored.Filled.ArrowBack,
                         contentDescription = "Back",
-                        modifier = Modifier.size(dimensions.iconSizeSmall),
+                        modifier = Modifier.size(AppDimensions.iconSizeSmall),
```

Comment:
Great that you noticed that the padding was only affecting the suggested text and not the written one too. Great job identifying and fixing this! It’s details like these that make the UI polished and professional. Keep it up!